src/source/PeerConnection/PeerConnection.c

```diff
@@ -714,6 +714,11 @@ STATUS createPeerConnection(PRtcConfiguration pConfiguration, PRtcPeerConnection
 
     NULLABLE_SET_EMPTY(pKvsPeerConnection->canTrickleIce);
 
+    if (!pConfiguration->kvsRtcConfiguration.disableSenderSideBandwidthEstimation) {
```

Comment:
Btw, missed asking. Are we enabling this by default anywhere?

Reply:
we zero-init `kvsRtcConfiguration`
that means `disableSenderSideBandwidthEstimation` will be set to false, 
therefore it is enabled by default

Reply:
Just saw it. Perfect :)

Reply:
i think it's a good idea to enable it by default, that's what chrome does